versions.gradle

```diff
@@ -0,0 +1,154 @@
+/**
+ * Shared file between builds so that they can all use the same dependencies and
+ * maven repositories.
+ **/
+ext.deps = [:]
```

Comment:
We don't really have multiple modules in this sample repository, but the idea is great.
You might want to follow the structure like in [quality-matters/dependencies.gradle](https://github.com/artem-zinnatullin/qualitymatters/blob/47e484b691da83e66d7446be6344e8966d64f784/dependencies.gradle).
Also, as it's `Groovy` I would stick to camelCase rather than underscores.

Reply:
sure. I'll do those updates, to be honest I borrowed this style from the google samples of architecture components, I can do that clean up :)